src/test/java/pl/szczesniak/dominik/whattowatch/movies/domain/MoviesToWatchServiceTest.java

```diff
@@ -184,10 +185,91 @@ void list_should_be_empty_when_no_movie_added() {
 		userProvider.addUser(userId);
 
 		// when
-		final List<Movie> movies = tut.getList(userId);
+		final List<Movie> movies = tut.getMoviesToWatch(userId);
 
 		// then
 		assertThat(movies.isEmpty()).isTrue();
 	}
 
+	@Test
+	void should_add_movie_to_watched_list_and_remove_from_movies_to_watch_list() {
+		// given
+		final UserId userId = new UserId(1);
+		userProvider.addUser(userId);
+
+		final MovieId parasite = tut.addMovieToList(new MovieTitle("Parasite"), userId);
+		final MovieId starWars = tut.addMovieToList(new MovieTitle("Star Wars"), userId);
+		final MovieId viking = tut.addMovieToList(new MovieTitle("Viking"), userId);
+
+		// when
+		tut.moveMovieToWatchedList(starWars, userId);
+
+		// then
+		assertThat(tut.getMoviesToWatch(userId)).hasSize(2)
+				.extracting(Movie::getMovieId, Movie::getTitle)
+				.containsOnly(
+						tuple(parasite, new MovieTitle("Parasite")),
+						tuple(viking, new MovieTitle("Viking"))
+				);
+
+		assertThat(tut.getWatchedMovies(userId)).hasSize(1)
+				.extracting(WatchedMovie::getMovieId, WatchedMovie::getTitle)
+				.containsOnly(tuple(starWars, new MovieTitle("Star Wars")));
+	}
+
+	@Test
+	void should_throw_exception_when_trying_to_move_the_movie_when_user_does_not_exist() {
+		// when
+		Throwable thrown = catchThrowable(() -> tut.moveMovieToWatchedList(new MovieId(1), new UserId(123)));
+
+		// then
+		assertThat(thrown).isInstanceOf(UserDoesNotExistException.class);
+	}
+
+	@Test
+	void should_throw_exception_when_trying_to_move_the_movie_when_userid_does_not_match_movie_id() {
```

Comment:
nazwa testu chyba skopana, ma rzucić wyjątek gdy próbujemy przenieś film (gdzie??) gdy userid nie pasuje do movieId (dlaczego userid mialoby pasowac do movieid? to chyba nie o to tu chodzi, przemyśl sobie)